CHANGELOG.md

```diff
@@ -5,4 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2024-11-29
+
+### Changes
+
+Added a `docs` folder to hold the
+[Canonical Sphinx starter pack](https://github.com/canonical/sphinx-docs-starter-pack)
+and to eventually publish the docs on Read the Docs.
```

Comment:
Any dated entries should go below `unreleased`. The reason we have unreleased is because for code changes to take effect, we have to do a release to PyPI. I understand these doc changes will go in straight away so you can make a dated entry from the PR

Reply:
Thanks for explaining -- I will move the entry under `unreleased`